game.py

```diff
@@ -530,6 +686,36 @@ def _circle(self, r, cx, cy):
     def _footer(self):
         return '</svg>\n'
 
+    def save_highscore(self):
+        file_path = os.path.join(self._activity_root, 'data', 'highscore')
+        logging.debug(file_path)
+        highscore = []
+        highscore.append(0)
+        if os.path.exists(file_path):
+            File = open(file_path, "r")
+            highscore = File.readlines()
+            File.close()
+
+        int_highscore = int(highscore[0])
+        if not (int_highscore > self._correct):
+            File = open(file_path, "w")
+            File.write(str(self._correct))
+            File.close()
+
+
+    def load_highscore(self):
+        file_path = os.path.join(self._activity_root, 'data', 'highscore')
+        logging.debug(file_path)
+        if os.path.exists(file_path):
+            try:
+                File = open(file_path, "r")
+                highscore = int(File.readlines()[0])
+                File.close()
+                return highscore
+            except BaseException:
+                return 0
+        else:
+            return 0
```

Comment:
Can be compressed to 
```suggestion
        return 0
```